keyboards/3.0/en-t-k0-teletype.xml

```diff
@@ -33,13 +33,13 @@
       <row keys="Q W E R T Y U I O P gap2" />
       <row keys="gap A S D F G H J K L enter gap" />
       <row keys="figs Z X C B V N M ltrs linefeed null" />
-      <row keys="gap space gap" stretch="space" />
+      <row keys="gap space gap" />
```

Comment:
Do we indicate anywhere that the default `space` has the `stretch` property in some way?  We're removing the row-attribute here, but I don't see where we add something to indicate why we don't need to specify it here.